cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -12,6 +12,26 @@ WORKER_NODE_ROOT="${CLUSTER_NAME}-worker"
 
 OPERATOR_GIT_HASH=8d3c30de8ec5a9a0c9eeb84ea0aa16ba2395cd68  # release-4.4
 
+function ensure_k8s_object {
```

Comment:
How about `wait_for_k8s_object` ?

Reply:
I will change all 'ensure' to 'wait' because those functions dont  act when what need to be ensured is missing.